src/styles/main.scss

```diff
@@ -1,3 +1,44 @@
 body {
   margin: 0;
+  counter-reset: my-counter;
```

Comment:
use utils folder for `variables.scss`, `normalize.scss`
BEM blocks for diffrent sections 
refer the videos again

Reply:
done.